### PR TITLE
Update the multi-schema change DDL limitation

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Defined2014 — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-alter-table.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-- 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
+- 使用 `ALTER TABLE` 语句进行多个模式更改时，不允许在多个更改中指定同一个模式对象。
```
Comment (tangenta) [inline sql-statements/sql-statement-alter-table.md:157]:
```suggestion
- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。
```

[the remark this answers is not in the document]
Comment (tangenta) [reply sql-statements/sql-statement-alter-table.md]: Updated.

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
status LGT2

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: mysql-compatibility.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ MySQL 系统变量 `optimizer_switch` 在 TiDB 中是只读的，对查询计划
 
 TiDB 中，所有支持的 DDL 变更操作都是在线执行的。与 MySQL 相比，TiDB 中的 DDL 存在以下限制：
 
-* 不能在单条 `ALTER TABLE` 语句中完成多个操作。例如，不能在单个语句中添加多个列或索引，否则，可能会输出 `Unsupported multi schema change` 的错误。
+* 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。例如，`ALTER TABLE t1 MODIFY COLUMN c1 INT, DROP COLUMN c1` 在两个更改中都指定了 `c1` 列，执行该语句会输出 `Unsupported operate same column/index` 的错误。
```
Comment (seiya-annie) [inline mysql-compatibility.md:108]: 1. No Support TiDB-specific Multi-Schema Change like ADD TIFLASH REPLICA, ADD PARTITION, ALTER PARTITION, etc.
2. the order of sub-ddl may be not fully compatible with MySQL.

File: sql-statements/sql-statement-alter-table.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 TiDB 中的 `ALTER TABLE` 语法主要存在以下限制：
 
-- 不支持在单个 `ALTER TABLE` 语句中进行多个更改。
+- 使用 `ALTER TABLE` 语句修改一个表的多个模式对象（如列、索引）时，不允许在多个更改中指定同一个模式对象。
```
Comment (seiya-annie) [inline sql-statements/sql-statement-alter-table.md:157]: tidb check consistency with the schema before ddl. if some schema is added in the ddl, can't alter it.eg.
`ALTER TABLE ADD INDEX i(b), DROP INDEX i;

**Review by seiya-annie — COMMENTED** [review]
add some comments, others LGTM

**Pull request merged**